.changeset/eight-flies-cross.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
You need to remove the changesets (or to wait after the main is released and merge it then), otherwise the the would come up an alpha release of the cli, sidebar-settings and guideline-blocks-settings. Which is not needed at all

Reply:
done